packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -152,6 +153,27 @@ export class Meter implements types.Meter {
     this._metrics.set(name, metric);
   }
 
+  /**
+   * Provide a pre-computed re-useable LabelSet by
+   * converting the unordered labels into a canonicalized
+   * set of lables with an unique identifier, useful for pre-aggregation.
+   * @param labels user provided unordered Labels.
+   */
+  labels(labels: types.Labels): types.LabelSet {
+    let keys = Object.keys(labels).sort();
+    let identifier = keys.reduce((result, key) => {
```

Comment:
Does this imply that we don't support `,` and `:` in metric label values? Do we enforce that validation anywhere and document it?